automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -149,10 +149,11 @@ trait GPAllocBeforeAndAfterAll extends GPAllocUtils with BeforeAndAfterAll {
           case Right(_) =>
             IO(logger.info(s"Created initial runtime ${project.value} / ${initalRuntimeName.asString}"))
           case Left(err) =>
-            IO(logger.warn(
-                 s"Failed to create initial runtime ${project.value} / ${initalRuntimeName.asString} with error"
-               ),
-               err)
+            IO(
```

Comment:
Before this was `IO(logger.warn(msg), err)` of type `IO[(Unit, Throwable)]`.

Should actually be passing the error to the logger, e.g. `IO(logger.warn(msg, err))` of type `IO[Unit]`.

I wish the Scala compiler caught this somehow 🤦